config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -34,7 +34,6 @@ presubmits:
         - --aws
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --build=bazel
-        - --check-leaked-resources=false
```

Comment:
(this is false by default: https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L130)